server/mvcc/backend/backend.go

```diff
@@ -446,6 +455,8 @@ func (b *backend) defrag() error {
 		b.lg.Info(
 			"defragmented",
 			zap.String("path", dbp),
+			zap.String("freelistType", string(options.FreelistType)),
+			zap.Int("initMmapSize", options.InitialMmapSize),
```

Comment:
Can we make this struct field name consistent?

`init-mmap-size`?

Reply:
good point